files/scripts/sonic-cfggen

```diff
@@ -0,0 +1,11 @@
+NS="asic" #Network namespace
```

Comment:
This line appears multiple times in this PR.
ASIC and namespaces should be independent concepts. I see in currently implementation we have one ASIC per namespace, but this should be treated as one choice and one specific configuration in a multiple-namespaces flexibility.

Configuration should appear once.

Reply:
Added a new script sonic-netns-exec - to execute any command in a given namespace. Removed usage of different wrappers for sonic-db-cli and sonic-cfggen.